addons/remove-default-flyout/addon.json

```
@@ -0,0 +1,20 @@
+{
+  "name": "Remove default values in flyout",
+  "description": "todo",
```

My proposed name and description:
```suggestion
  "name": "Remove default block values",
  "description": "Removes the default values of text and number inputs in blocks, leaving them blank.",
```